concourse/pipelines/rhui-release.yaml

```
@@ -0,0 +1,175 @@
+jobs:
+- name: manual-trigger
+  plan:
+  - get: cds-image
+    trigger: false
+  - get: rhua-image
+    trigger: false
+- name: deploy-staging-us-west1
+  plan:
+  - get: cds-image
+    passed: [ ]
+    trigger: true
+  - get: rhua-image
+    passed: [ ]
+    trigger: true
+  - config:
+      image_resource:
+        source:
+          repository: google/cloud-sdk
+          tag: alpine
+        type: registry-image
+      platform: linux
+      run:
+        args:
+        - -exc
+        - gcloud compute instance-groups managed rolling-action replace --quiet rhua-mig-staging-us-west1 --region=us-west1 --project=google.com:rhel-infra; gcloud compute instance-groups managed wait-until --version-target-reached --quiet rhua-mig-staging-us-west1 --region=us-west1 --project=google.com:rhel-infra;
+        path: sh
```

doesn't seem we need the shell here, we're not doing any substitutions. suggest:
```suggestion
        - compute
        - instance-groups
        - managed
        - rolling-action
        - replace
        - --quiet
        - rhua-mig-staging-us-west1
        - --region=us-west1
        - --project=google.com:rhel-infra
        path: gcloud
```

and a second task for the wait action, configured the same way